business_requirement_deliverable_cost/models/business.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class BusinessRequirementResource(models.Model):
+    _inherit = "business.requirement.resource"
+
+    sale_price_unit = fields.Float(
+        string='Sales Price',
+        readonly=True,
+        groups='business_requirement_deliverable_cost.\
+group_business_requirement_estimation',
+    )
+    sale_price_total = fields.Float(
+        compute='_compute_sale_price_total',
+        string='Total Revenue',
+        groups='business_requirement_deliverable_cost.\
+group_business_requirement_estimation',
+    )
+    unit_price = fields.Float(
+        groups='business_requirement_deliverable_cost.\
+group_business_requirement_cost_control',
+    )
+    price_total = fields.Float(
+        groups='business_requirement_deliverable_cost.\
+group_business_requirement_cost_control',
+    )
+
+    @api.multi
+    @api.depends('sale_price_unit', 'qty')
+    def _compute_sale_price_total(self):
+        for resource in self:
+            resource.sale_price_total = resource.sale_price_unit * resource.qty
```

Comment:
@elicoidal  the conversion will be taken into account when calculating `sale_price_unit`.